structure/heap/heap_test.go

```diff
@@ -0,0 +1,100 @@
+package heap
```

Comment:
```suggestion
package heap_test
```

Prefer black box testing for tests that are only testing the exported API.